raiden_contracts/tests/utils.py

```diff
@@ -1,56 +1,124 @@
+    min_expiration_delta = min_expiration_delta or (len(unlockable_amounts) + 1)
+    max_expiration_delta = max_expiration_delta or (min_expiration_delta + SETTLE_TIMEOUT_MIN)
+    unlockable_locks = [
+        [
+            current_block + random.randint(min_expiration_delta, max_expiration_delta),
+            amount,
+            *random_secret()
+        ]
+        for amount in unlockable_amounts
+    ]
+    expired_locks = [
+        [current_block, amount, *random_secret()]
+        for amount in expired_amounts
+    ]
+    return (unlockable_locks, expired_locks)
 
 
-def check_channel_closed(channel_identifier, closing_participant):
-    def get(event):
-        assert event['args']['channel_identifier'] == channel_identifier
-        assert event['args']['closing_participant'] == closing_participant
-    return get
+def get_pending_transfers_tree(
+        web3,
+        unlockable_amounts=[],
+        expired_amounts=[],
+        min_expiration_delta=None,
+        max_expiration_delta=None
+):
+    types = ['uint256', 'uint256', 'bytes32']
+    (unlockable_locks, expired_locks) = get_pending_transfers(
+        web3,
+        unlockable_amounts,
+        expired_amounts,
+        min_expiration_delta,
+        max_expiration_delta
+    )
 
+    pending_transfers = unlockable_locks + expired_locks
 
-def check_channel_unlocked(channel_identifier, payer_participant, transferred_amount):
-    def get(event):
-        assert event['args']['channel_identifier'] == channel_identifier
-        assert event['args']['payer_participant'] == payer_participant
-        assert event['args']['transferred_amount'] == transferred_amount
-    return get
+    hashed_pending_transfers = [
+        Web3.soliditySha3(types, transfer_data[:-1])
+        for transfer_data in pending_transfers
+    ]
 
+    hashed_pending_transfers, pending_transfers = zip(*sorted(zip(
+        hashed_pending_transfers,
+        pending_transfers
+    )))
 
-def check_transfer_updated(channel_identifier, closing_participant):
-    def get(event):
-        assert event['args']['channel_identifier'] == channel_identifier
-        assert event['args']['closing_participant'] == closing_participant
-    return get
+    merkle_tree = compute_merkle_tree(hashed_pending_transfers)
+    packed_transfers = get_packed_transfers(pending_transfers, types)
 
+    return PendingTransfersTree(
+        transfers=pending_transfers,
+        unlockable=unlockable_locks,
+        expired=expired_locks,
+        packed_transfers=packed_transfers,
+        merkle_tree=merkle_tree,
+    )
 
-def check_channel_settled(channel_identifier):
-    def get(event):
-        assert event['args']['channel_identifier'] == channel_identifier
-    return get
+
+def get_packed_transfers(pending_transfers, types):
+    packed_transfers = [encode_abi(types, x[:-1]) for x in pending_transfers]
+    return reduce((lambda x, y: x + y), packed_transfers)
+
+
+def get_settlement_amounts(
```

Comment:
A small docstring explaining what this helper function returns would be helpful here